src/draw/drawRectangle.ts

```diff
@@ -56,14 +56,16 @@ export function drawRectangle(
   options: DrawRectangleOptions<Mask | Image> = {},
 ): Image | Mask {
   const {
+    width: rectangleWidth = image.width,
+    height: rectangleHeight = image.height,
     origin = { column: 0, row: 0 },
-    width = image.width,
-    height = image.height,
     strokeColor = getDefaultColor(image),
     fillColor = 'none',
   } = options;
-  const { column, row } = origin;
-
+  const width = Math.round(rectangleWidth ?? image.width);
```

Comment:
rectangleWidth should always be defined and set to the default value image.width so `?? image.width` seems useless.